File: level1/Python/소수만들기.py
def solution(nums):
    answer = 0
    
    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            for k in range(j+1, len(nums)):
                sub = nums[i]+nums[j]+nums[k]
                for z in range(2, sub):
                    if sub % z == 0:
                        break
                else:
                    answer+=1
    

    return answer

File: level1/C++/소수만들기.cpp
#include <vector>
#include <iostream>
using namespace std;

int solution(vector<int> nums) {
    int answer = -1;

    for(int i=0;i<nums.size();i++)
    {
        for(int j=i+1;j<nums.size();j++)
        {
            for(int k=j+1;k<nums.size();k++)
            {
                int sub = nums[i] + nums[j]+ nums[k];
                bool check = true;
                for (int z = 2;z<sub;z++){
                    if(sub%z==0){
                        check = false;
                        break;
                    }
                }
                if(check){
                    answer +=1;
                }
            }
        }
    }
    

    return answer+1;
}
